test/performance/README.md

```diff
@@ -0,0 +1,82 @@
+# Kueue Performance Testing
+
+## Measurements
+
+### Job startup latency
+
+How fast do jobs transition from `created` to `started` state?
+Time spent between the transition from `job.CreationTimestamp.Time` to `job.Status.StartTime.Time` state.
+
+High Job startup latency in Kueue is expected when the total quota is not enough to schedule all jobs immediately, because the jobs need to queue.
+
+### Job startup throughput
+
+The best workload admission rate per second within 1 minute intervals. 
+The rate is measured every 5 seconds (see more details in [PromQL examples](https://prometheus.io/docs/prometheus/latest/querying/examples/#subquery)):
+
+`max_over_time(sum(rate(kueue_admitted_workloads_total{cluster_queue="{{$clusterQueue}}"}[1m]))[{{$testTimeout}}:5s])`
+
+This measurement is not efficient if the cluster quota is big enough to schedule all workloads of the test immediately, because Kueue immediately admits all the workloads and the `kueue_admitted_workloads_total` never increases. In this case, the PromQL query returns 0.
+## How to run the test?
+
+### Prerequisites
+
+1. Deploy [Kueue](https://github.com/kubernetes-sigs/kueue/blob/main/docs/setup/install.md)
+2. Make sure you have `kubectl`, [jq](https://stedolan.github.io/jq/download/), [golang version](https://github.com/mikefarah/yq) of `yq` and `go` 
+3. Checkout `Clusterloader2` framework: https://github.com/kubernetes/perf-tests and build `clusterloader` binary:
+
+    * change to `clusterloader2` directory
+    * run `go build -o clusterloader './cmd/'` 
+
+### Run the test
+
+1. Copy an environment file example to `.env` file: 
+
+    * `cp .env.example .env`
+    
+2. Configure the environment variable file:
+
+```
+# Clusterloader home directory (checkout https://github.com/kubernetes/perf-tests)
+export CL2_HOME_DIR="/Users/johny/perf-tests/clusterloader2"
+
+# Run the performance test with Kueue (this requires Kueue to be pre-deployed to the cluster)
+# or without Kueue
+export USE_KUEUE=false
+
+# Test iterations:
+# number-of-small-jobs number-of-large-jobs job-replica-running-time test-timeout cluster-queue-CPU-quota cluster-queue-memory-quota
+export EXPERIMENTS=(
+    "10 2 0 2s 3m 100 100Gi"
+    "20 2 0 2s 5m 100 100Gi"
+)
+
+# Kubeconfig file location
+export KUBECONFIG="$HOME/.kube/config"
+
+# Kubernetes kind
+export PROVIDER="gke"
+
+```
```

Comment:
let's not copy this here to avoid duplication

Reply:
Now it's a table with variables and corresponding descriptions.